spec/models/form526_submission_spec.rb

```diff
@@ -98,72 +98,87 @@
   end
 
   describe 'scopes' do
-    describe 'pending_backup_submissions' do
-      let!(:new_submission) { create(:form526_submission) }
-      let!(:rejected_backup_submission) do
-        create(:form526_submission, :backup_path, backup_submitted_claim_status: 'rejected')
-      end
-      let!(:accepted_backup_submission) do
-        create(:form526_submission, :backup_path, backup_submitted_claim_status: 'accepted')
-      end
-      let!(:backup_submission_a) { create(:form526_submission, :backup_path) }
-      let!(:backup_submission_b) { create(:form526_submission, :backup_path) }
+    let!(:in_process) { create(:form526_submission) }
+    let!(:expired) { create(:form526_submission, :created_more_than_3_days_ago) }
+    let!(:happy_path_success) { create(:form526_submission, :with_submitted_claim_id) }
+    let!(:pending_backup) { create(:form526_submission, :backup_path) }
+    let!(:accepted_backup) { create(:form526_submission, :backup_path, :backup_accepted) }
+    let!(:rejected_backup) { create(:form526_submission, :backup_path, :backup_rejected) }
+    let!(:remediated) { create(:form526_submission, :remediated) }
+    let!(:remediated_and_expired) { create(:form526_submission, :remediated) }
```

Comment:
Should this also include the `created_more_than_3_days_ago` trait?

Reply:
yes good catch